tutorials/docker/startup.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+Xvfb :0 -screen 0 800x600x24+32 &
+DISPLAY=:0 gradle run
```

Comment:
There is a Gradle wrapper in this PR, which means that we can do: `gradlew run`. And possibly `chmod +x gradlew`.

Reply:
I have removed Gradle's wrapper. It makes Docker image rebuilds slower because it downloads Gladle on every change in the project files.